src/app/find-help/reply-offer/reply-offer.component.ts

```diff
@@ -1,11 +1,13 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
 import { defaults } from '@app/shared/utils';
 import { SendMessageDTO, MessageResourceService } from '@app/core/api';
 import { SnackbarService } from '@app/shared/services';
 import { ALERT_TYPES } from '@app/shared/models';
-import { switchMap, take } from 'rxjs';
-import { ReCaptchaV3Service } from 'ng-recaptcha';
+import { take } from 'rxjs';
```

Comment:
Should t it import it from rxjs/operators I am not sure, here it may import probably  the whol library again but I am not sure If is been change in Rxjs over time

Reply:
done